src/modals/Modal.tsx

```diff
@@ -100,6 +105,7 @@ export class Modal extends React.PureComponent<ModalProps> {
                             ClassNames.MODAL_DIALOG, //prettier
                             size && size,
                         ])}
+                        style={size === ModalSize.CUSTOM ? {width: `${width}px`, height: `${height}px`} : {}}
```

Comment:
Please add option to use specified styles as well

Reply:
We dont want user to send this custom style from components, so not adding support for style props.  Discussed with @akate1  already.